packages/main/src/themes/sap_belize/Panel-parameters.css

```diff
@@ -3,6 +3,7 @@
 :root {
 	--_ui5_panel_header_padding_right: 1rem;
 	--_ui5_panel_header_height: 3rem;
+	--_ui5_panel_header_background_color: var(--sapList_HeaderBackground);
```

Comment:
Looking into the specification, for Belize I think that sapUiPageHeaderBackground is the correct parameter , as well I didn't see --sapList_HeaderBackground specified anywhere, please check.